lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -576,12 +576,15 @@ void SessionStateEnforcer::prepare_response_to_access(
   /* AMBR value need to compared from AMF and PCF, then fill the required
    * values and sent to AMF.
    */
-  // For now its default QOS, AMBR has default values
+  // For now its default QOS, AMBR is 1 Gbps downlink
   rsp->mutable_downlink_unit_type()->set_ambr_unit_type(
-      magma::AmbrUnit::Kbps_64);
-  rsp->mutable_uplink_unit_type()->set_ambr_unit_type(magma::AmbrUnit::Kbps_64);
+      magma::AmbrUnit::Gbps_1);

```

Comment:
see my comment at the bottom first:

in this case if we want to set any ambrunit we will need to create a switch function that transforms the  QOS value (for example 11) into `magma::AmbrUnit::Gbps_1`.  If you don't define any enum you can just do 

```
 rsp->mutable_downlink_unit_type()->set_ambr_unit_type(req->qos->get_ambr_unit_type);

```
(note req object is made up)

Reply:
Right now for 5G context,  sessiond is using a default value. As part of QoS changes this value will be derived from either default subscriber profile or values stored in policydb.  The qos infrastructure like "qos.get_ambr_unit_type" still needs to be updated for 5G Context in sessiond.